src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", Helvetica, sans-serif;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: $Main_text-color;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    line-height: 14px;
+    font-size: 10px;
+    color: $Secondary_text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    .card__reviews {
+      align-self: center;
+      font-size: 10px;
+      color: $Main_text-color;
+    }
+
+    .card__stars {
+      display: inline-block;
+    }
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      color: $Secondary_text-color;
+    }
+
+    &-amount {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $Main_text-color;
+    }
+  }
+
+  &__button {
+    height: 40px;
+    padding: 12px 70px;
```

Comment:
Not quite correct to set such large left/right padding, because if there is more text in the button, it will look strange. More correctly use text-align: center for example
![image](https://github.com/mate-academy/layout_product-cards/assets/54936044/57243dc9-a716-4e40-a15d-9fed9c4103ed)